src/pro_create.cpp

```diff
@@ -243,6 +243,9 @@ struct obj_data *can_program(struct char_data *ch)
       comp = NULL;
     if (!comp) {
       send_to_char(ch, "There is no computer here for you to use.\r\n");
+      // A computer isn't used to design a complex form, but we use a different command for that
+      if (PRF_FLAGGED(ch, PRF_SEE_TIPS) && IS_OTAKU(ch))
+        send_to_char("[OOC: To learn a complex form, try ^WFORMS LEARN <FORM>^n.]\r\n", ch);
```

Comment:
Does this check need to be outside of the `!comp` check then?

Reply:
It needs to be inside an error condition, and the ones I think an otaku would reach when attempting "design <form>" is either the absence of a computer here, or the absence of <form> in a present computer.